frontend/src/shared/pages/Home.vue

```diff
@@ -28,6 +31,7 @@
 
 <script lang="ts" setup>
 import { ref } from 'vue'
+import AxiosLineTemp from "@/shared/AxiosLineTemp.vue";
 
```

Comment:
well this has to be aligned with the main branch: This would break the splash screen and routing to the CardsDetail:
Is it temporary - so not to be merged into main? or is it something we could copy into test folder as well?

Reply:
will move it to the temptest folder and remove it from main view - it was left here only temporary to check how axios behave after your kaatest  changes